README.md

```diff
@@ -269,7 +283,7 @@ It is recommended to read the manpage 'apt_preferences(5)'
   * 'content': See `content` in `apt::key`
   * 'source': See `source` in `apt::key`
   * 'options': See `options` in `apt::key`
-* `pin`: See apt::pin. Defaults to false.
+* `pin`: See apt::pin. Defaults to undef. Can be a string, number, or a hash to be passed to `create\_resources` for `apt::pin`.
```

Comment:
I wouldn't mention the implementation detail of `create_resources`, just that it will be passed as parameters to `apt::pin`.